bentoml/_internal/frameworks/keras.py

```diff
@@ -317,7 +317,14 @@ def _run_method(
 
             with tf.device(runnable_self.device_name):
                 params = params.map(_mapping)
-                res: "tf_ext.EagerTensor" | "ext.NpNDArray" = raw_method(params.args)
+                if len(params.args) == 1:
+                    res: "tf_ext.EagerTensor" | "ext.NpNDArray" = raw_method(
+                        *params.args
+                    )
+                else:
+                    res: "tf_ext.EagerTensor" | "ext.NpNDArray" = raw_method(
+                        params.args
+                    )
```

Comment:
Maybe a nicer way is to make the params.args to be a list
```python
if len(params.args) == 1:
    args = [params.args]
else:
    args = params.args

res = raw_method(*args)
```

Reply:
the intention logic is slightly different but I made a similar change